crates/matrix-sdk-ui/src/room_list_service/room.rs

```diff
@@ -29,12 +30,17 @@ use crate::{
 /// A room in the room list.
 ///
 /// It's cheap to clone this type.
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub struct Room {
     inner: Arc<RoomInner>,
 }
 
-#[derive(Debug)]
+impl fmt::Debug for Room {
+    fn fmt(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
+        formatter.debug_tuple("Room").field(&self.id().to_owned()).finish()
```

Comment:
nit: do we need `&{something}.to_owned()`?

```suggestion
        formatter.debug_tuple("Room").field(self.id()).finish()
```

Reply:
Yeah… otherwise `&RoomId` doesn't have a size known at compile-time.